src/workflow.ts

```diff
@@ -71,7 +71,9 @@ export interface WorkflowContext extends DBOSContext {
   getEvent<T extends NonNullable<any>>(workflowUUID: string, key: string, timeoutSeconds?: number): Promise<T | null>;
   retrieveWorkflow<R>(workflowUUID: string): WorkflowHandle<R>;
 
-  sleep(durationSec: number): Promise<void>;
+  sleepsecs(durationSec: number): Promise<void>;
+  sleepms(durationMS: number): Promise<void>;
+  sleep(durationSec: number): Promise<void>; // Deprecated
```

Comment:
Why not just leave sleep the way it is and introduce `sleepms`? This way we don't have a breaking change.

Reply:
I did leave sleep the way it is and introduce sleepms so that it is not a breaking change.  But sleep means ms everywhere else in the world, that's why deprecate.  Removing the deprecated one would be a breaking change, but how many of those do you think are actually out there?  Is sleeping something our customers do a lot of?

Reply:
I think sleeping is going to be a big use case for workflow orchestration.

Unclear what are the semantics for sleeping in the world: python, bash and C uses seconds by default. Go asks you for an explicit `Duration` object. JS/node setInterval is in ms.

Reply:
This is exactly why I wanted to clarify it.  (In absence of a duration object, which is what C++ uses also.)

Reply:
Java uses ms, or TimeUnit.

Reply:
And, clearly, we weren't sure, because we went with the self-contradiction approach.

Reply:
I think having `sleep` and `sleepms` is enough of an indicator that `sleep` is *not* milliseconds. Seconds feels like the next most likely duration unit after milliseconds + user's get IDE tool tip hints + docs. changing `sleep` -> `sleepsecs` seems like overthinking

Reply:
I also like keeping `sleep` the way it is (seconds) and add `sleepms`

Reply:
Very well, but you are all _wrong_.
![image](https://github.com/dbos-inc/dbos-transact/assets/134347445/64abaac1-938f-4093-917e-f6d59b2ebd50)

Reply:
Arguably, the right thing here would be to make sleep be milliseconds across the board. Maybe file an issue to track making a breaking change whenever we get to v2.x?

Reply:
> user's get IDE tool tip hints

When writing code, yes.  When reviewing code, I often do not.

Reply:
> When writing code, yes. When reviewing code, I often do not.

All the more reason to file an issue to track making the breaking change

Reply:
How about adding an optional param to sleep that denotes the unit?
The breaking change will be to change the default value from `min` to `ms`
Adding the `sleepms` is a mistake I think as it will have to be removed later on. 
All javascript developers expect sleep to be in ms since it is almost always implemented using  `setTimout`.